com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-properties/advanced-options/geometry-type.md

```diff
@@ -0,0 +1,4 @@
+<tr>
+<td>**Geometry Type**</td>
+<td>Indicates the type of geometry being used to represent the hair, allowing the shading model to make informed approximations. Note that switching between these two types will produce slightly differing results, due to how the underling shading model approximates the hair fiber. The options are:<br/>&#8226; **Cards**: Representation by hair "cards"; a projection of hair fibers onto a drastically simplified proxy geometry.<br/>&#8226; **Strands**: Representation by screen-aligned ribbons, or tubes; an explicit representation of a hair fiber.</td>
```

Comment:
<td>Indicates what geometry is used to represent the hair, which allows the shading model to make informed approximations. The shading model approximates the hair fiber differently in each of the following representations: <br/>&#8226; **Cards**:  Hair fibers are projected onto simplified proxy geometry called "cards".<br/>&#8226; **Strands**: Hair fibers are represented by screen-aligned ribbons or tubes.</td>